packages/plugin-lighthouse/src/lib/utils.ts

```diff
@@ -68,3 +75,89 @@ export function validateOnlyAudits(
   }
   return true;
 }
+
+type UnsupportedDetail =
+  | Details.CriticalRequestChain
+  | Details.List
+  | Details.TreemapData
+  | Details.Screenshot
+  | Details.Filmstrip
+  | Details.DebugData;
+type UnsupportedDetailTypes = UnsupportedDetail['type'];
+
+export function toAuditOutputs(
+  lhrAudits: Record<string, Result>,
```

Comment:
Will the key ever be relevant for this function? If not, wouldn't it be easier to just pass the values to this function?

```suggestion
  lhrAudits: Result[],
```